Resources/Prototypes/Entities/Structures/Power/Generation/PA/control_box.yml

```diff
@@ -15,6 +15,8 @@
   - type: ActivatableUI
     key: enum.ParticleAcceleratorControlBoxUiKey.Key
   - type: ActivatableUIRequiresPower
+  - type: AccessReader
+    access: [["ChiefEngineer"]]
```

Comment:
You could consider adding the Captain here, as Captains need engineering time and also have a direct interest in both a) making the station have power and b) the station not exploding

Reply:
Let me test again... but I think captain will have access to this either way .

Reply:
it's fine. Captain ID have access here.